index.html

```diff
@@ -196,6 +211,8 @@ <h2 class="title-about">About<br>Myself</h2>
         </li>
        </ul>
        <div><button class="sec-button">Get my resume</button></div>
+      </div>
+       <div class="new-lang">
        <div class="accordion"><p>Language</p><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="icon">
         <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
```

Comment:
- [ ] As requested by the previous reviewer please make sure that the color and font match the design. See image below for reference
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/194519437-6f406a94-8b1c-4d9b-8a45-5d6f5c68fb55.png)
### Your design
![your](https://user-images.githubusercontent.com/81727988/194519484-bbc9b195-950f-4b60-b27d-51c63880328c.png)